ipyleaflet/leaflet.py

```diff
@@ -782,6 +782,79 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class ImageService(Layer):
+    """ImageService class
+
+    Image Service layer for raster data served through a web service
+
+    Attributes
+    ----------
+    url: string, default ""
+        Url to the image service
+    f: string, default "image"
+        response format (use `'image'` to stream as bytes)
+    format: string, default "jpgpng"
+        format of exported image
+    pixel_type: string, default "UNKNOWN"
+        data type of the raster image
+    no_data: list, default []
+        pixel value or comma-delimited list of pixel values representing no data
+    no_data_interpretation: string, default ""
+        how to interpret no data values
+    interpolation: string, default ""
+        resampling process for interpolating the pixel values
+    compression_quality: int, default 100
+        lossy quality for image compression
+    band_ids: List, default []
+        Order of bands to export for multiple band images
+    time: List, default []
+        time instance or extent for image
+    rendering_rule: dict, default {}
+        rules for rendering
+    mosaic_rule: dict, default {}
+        rules for mosaicking
+    transparent: boolean, default False
+        If true, the image service will return images with transparency
+    endpoint: str, default 'Esri'
+        Endpoint format for building the export image url
+    attribution: string, default ""
+        Image service attribution.
+    crs: dict, default ipyleaflet.projections.EPSG3857
+        Projection used for this image service.
+    interactive: bool, default False
+        Emit when clicked or hovered
+    update_interval: int, default 200
+        Update interval for panning
```

Comment:
What is updated? What is the unit of this value?

Reply:
Great point. I expanded the description here. This is using `L.Toggle` with units in ms